task3/index.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Gektor</title>
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+        <body>
+            <div class="leftcol">
+                <header>
+                    <img class="logo" src="logo.jpg" align="center" alt="logotip"> <strong>Gektor</strong>
+                    <p class="username"><strong>Anastasiya Zyatkova</strong></p>
+                </header>
+                <div id="menu">
+                    <p><img class="picbut" src="home.png" alt="home" align="center"><button class="menubut"><strong>Home</strong></button></p>
+                    <p><img class="picbut" src="profile.png" alt="profile" align="center"><button class="menubut"><strong>Profile</strong></button></p>
+                    <p><img class="picbut" src="logout.png" alt="log out" align="center"><button class="menubut"><strong>Log Out</strong></button></p>
+                </div>
+                <button class="note1"><strong>Note</strong></button>
+            </div>
+
+            <div class="rightcol">
+                <div class="searchtags">
+                    <form>
+                        <input class="edit" id="searchtags" type="search"  placeholder="Search by tags..">
+                    </form>
+                </div>
+                <div class="filtred">
+                    <form>
+                        <p class="filt"><strong>Filtred by...</strong></p>
```

Comment:
Again this is not important text.
Additionally regarding semantic abilities of html it is better to mark this into `<fieldset>`
then the text-label became a meaningful `<legend>` tag